docs/transaction-metrics.asciidoc

```diff
@@ -0,0 +1,93 @@
+[x-pack]
+[[transaction-metrics]]
+== Configure transaction metrics
+
+++++
+<titleabbrev>Transaction metrics</titleabbrev>
+++++
+
+When enabled, {beatname_uc} produces the transaction histogram metrics that are used to power the APM app.
+Shifting this responsibility from APM app to APM Server results in improved query performance.
```

Comment:
```suggestion
Shifting this responsibility from APM app to APM Server results in improved query performance and removes the need to store unsampled transactions.
```

Not sure how much details we want to get into here. Storage reductions are the main driver behind this, query time may not be affected much.